modules/sre-cluster-access.adoc

```diff
@@ -10,20 +12,277 @@ SRE access to {product-title}
+== SRE access through PrivateLink VPC endpoint service
+
+PrivateLink VPC endpoint service is created as part of the ROSA cluster creation.
+
+When you have a PrivateLink ROSA cluster, its Kubernetes API Server is exposed through a load balancer that can only be accessed from within the VPC by default. Red Hat site reliability engineering (SRE) can connect to this load balancer through a VPC Endpoint Service that has an associated VPC Endpoint in a Red Hat-owned AWS account. This endpoint service contains the name of the cluster, which is also in the ARN.
+
+Under the *Allow principals* tab, a Red Hat-owned AWS account is listed. This specific user ensures that other entities cannot create VPC Endpoint connections to the PrivateLink cluster's Kubernetes API Server.
+
+When Red Hat SREs access the API, this fleet management plane can connect to the internal API through the VPC endpoint service.
+
+[id="rosa-policy-rh-access_{context}"]
+= Red Hat support access
+Members of the Red Hat Customer Experience and Engagement (CEE) team typically have read-only access to parts of the cluster. Specifically, CEE has limited access to the core and product namespaces and does not have access to the customer namespaces.
+
+[cols= "2a,4a,4a,4a,4a",options="header"]
+
+|===
+
+| Role | Core namespace | Layered product namespace | Customer namespace | AWS account^*^
+
+|OpenShift SRE| Read: All
+
+Write: Very
+
+limited ^[1]^
+| Read: All
+
+Write: None
+| Read: None^[2]^
+
+Write: None
+|Read: All ^[3]^
+
+Write: All ^[3]^
+
+|CEE
+|Read: All
+
+Write: None
+
+|Read: All
+
+Write: None
+
+|Read: None^[2]^
+
+Write: None
+
+|Read: None
+
+Write: None
+
+|Customer administrator
+|Read: None
+
+Write: None
+
+|Read: None
+
+Write: None
+
+| Read: All
+
+Write: All
+
+|Read: All
+
+Write: All
 
+|Customer user
+|Read: None
+
+Write: None
+
+|Read: None
+
+Write: None
+
+|Read: Limited^[4]^
+
+Write: Limited^[4]^
+
+|Read: None
+
+Write: None
+
+|Everybody else
+|Read: None
+
+Write: None
+|Read: None
+
+Write: None
+|Read: None
+
+Write: None
+|Read: None
+
+Write: None
+
+|===
+--
+1. Limited to addressing common use cases such as failing deployments, upgrading a cluster, and replacing bad worker nodes.
+2. Red Hat associates have no access to customer data by default.
+3. SRE access to the AWS account is an emergency procedure for exceptional troubleshooting during a documented incident.
+4. Limited to what is granted through RBAC by the Customer Administrator and namespaces created by the user.
+--
+
+[id="rosa-policy-customer-access_{context}"]
+= Customer access
+Customer access is limited to namespaces created by the customer and permissions that are granted using RBAC by the Customer Administrator role. Access to the underlying infrastructure or product namespaces is generally not permitted without `cluster-admin` access. More information on customer access and authentication can be found in the "Understanding Authentication" section of the documentation.
```

Comment:
Change to: " For more information about customer access and authentication, see the "Understanding Authentication" section of the documentation.